docs/openapi-backend.yaml

```diff
@@ -0,0 +1,258 @@
+openapi: 3.0.3
+info:
+  title: IO Sign - Backend API
+  version: 0.0.1
+paths:
+  /signature-requests/{signatureRequestId}:
+    get:
+      operationId: getSignatureRequestById
+      summary: Get a Signature Request By Id
+      parameters:
+        - in: path
+          name: signatureRequestId
+          schema:
+            $ref: "#/components/schemas/SignatureRequestId"
+          required: true
+      responses:
+        "200":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureRequestDetailView"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /qtsp/clauses:
+    get:
+      operationId: getQtspClauses
+      summary: Get clauses list and MRC from QTSP
+      responses:
+        "200":
+          description: The QTSP clauses list
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QtspClauses"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        default:
+          $ref: "#/components/responses/Unexpected"
+  /signatures/{signatureId}:
+    get:
+      operationId: getSignatureById
+      summary: Get a Signature By Id
+      parameters:
+        - in: path
+          name: signatureId
+          schema:
+            $ref: "#/components/schemas/SignatureId"
+          required: true
+      responses:
+        "200":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureDetailView"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signatures:
+    post:
+      operationId: createSignature
+      summary: Create a Signature from SignatureRequest
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateSignatureBody"
+      responses:
+        "201":
+          description: Signature created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureDetailView"
```

Comment:
This call occurs when the user sends the signature request to the QTSP. Is it necessary that the response contain the SignatureDetailView? Or the status code is sufficient?